frontend/assets/stylesheets/_vars.scss

```diff
@@ -51,6 +51,12 @@ $mq-breakpoints: (
     mobileLandscape: gs-span(6) + $gs-gutter, // 480px
 );
 
+$max-widths: (
```

Comment:
This is baby steps towards abstracting the side-margins / layout code (so we can go to 1300px wide). Max widths are currently spread out across the code base. We need to standardise a bit before we can go wider.